css/style.css

```diff
@@ -0,0 +1,504 @@
+  display: flex;
+  align-items: center;
+}
+
+.content__bt__link {
+  text-decoration: none;
+  color: #fff;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 21px;
+  margin: 0 auto;
+}
+
+.content__img {
+  background-image: url(../img/big.png);
+  background-repeat: no-repeat;
+  height: 657px;
+  width: 770px;
+}
+
+/* footer */
+.footer {
+  width: 1888px;
+  margin: 0 auto;
+}
+
+.footer__scroll {
+  width: 32px;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+}
+
+.scroll-item {
+  background-color: #c8dbfb;
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+}
+
+@media (max-width: 1450px) {
+  .content__title {
+    font-size: 60px;
+  }
+}
+
+@media (max-width: 1366px) {
+  .wrap__site {
+    max-width: 1366px;
+  }
+
+  .header {
+    max-width: 960px;
+    margin: 35px auto 76px;
+  }
+
+  .header__menu {
+    max-width: 336px;
+  }
+
+  .menu__item {
+    font-size: 14px;
+    line-height: 16px;
+    padding: 7px 16px 0 0;
+  }
+
+  .content {
+    max-width: 927px;
+    height: 454px;
+    margin-bottom: 118px;
+    flex-direction: row;
+  }
+
+  .content__wrap {
+    height: 455px;
+    max-width: 612px;
+    margin-right: 0;
+  }
+
+  .content__title {
+    max-width: 427px;
+    height: 205px;
+    font-size: 50px;
+    line-height: 66px;
+  }
+
+  .content__desc {
+    max-width: 364px;
+    height: 43px;
+    font-size: 18px;
+    line-height: 21px;
+    margin-bottom: 40px;
+  }
+
+  .content__bt {
+    max-width: 209px;
+    height: 44px;
+  }
+  
+  .content__bt__link {
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  .content__img {
+    background-image: url(../img/small.png);
+    height: 439px;
+    width: 514px;
+    align-items: center;
+  }
+
+  .footer {
+    width: 1353px;
+  }
+}
+
+@media (max-width: 1170px) {
+  .wrap__site {
+    max-width: 1170px;
+  }
+
+  .header {
+    max-width: 85%;
+    margin: 31px auto 76px;
+  }
+
+  .header__menu {
+    width: 340px;
+  }
+
+  .menu__item {
+    padding: 5px 25px 0 0;
+  }
+  
+  .footer {
+    width: 1156px;
+  }
+}
+
+@media (max-width: 940px) {
+  .wrap__site {
+    max-width: 940px;
+    overflow-x: hidden;
+  }
+  
+  .header__menu,
+  .footer {
+    display: none;
+  }
+
+  .header__burger {
+    width: 35px;
+    display: flex;
+    cursor: pointer;
+  }
+
+  #header__toggle {
+    opacity: 0;
+  }
+
+  #header__toggle:checked ~ .header__btn > span {
+    transform: rotate(45deg);
+  }
+
+  #header__toggle:checked ~ .header__btn > span::before {
+    top: 0;
+    transform: rotate(0);
+  } 
+
+  #header__toggle:checked ~ .header__btn > span::after {
+    top: 0;
+    transform: rotate(90deg);
+  }
+
+  #header__toggle:checked ~ .header__menu {
+    visibility: visible;
+    right: 0;
+  }
+
+  .header__btn {
+    display: flex;
+    align-items: center;
+    position: fixed;
+    top: 20px;
+    right: 30px;
+    width: 35px;
+    height: 35px;
+    z-index: 1;
+    cursor: pointer;
+  }
+
+  .header__btn > span,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".header__btn > span" to come before selector "#header__toggle:checked ~ .header__btn > span" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/miniProject/pullRequest?prid=5593059)